### ISSUE TITLE
How to set option of plugin-proposal-pipeline-operator

### ISSUE THREAD

**Issue by oMaten** [babel/babel.github.io#1903]
After i add plugin `@babel/plugin-proposal-pipeline-operator`, there is no option choice for me.
The error hint as follow.
![image](https://user-images.githubusercontent.com/13043118/49148858-84bae780-f343-11e8-8543-6eabe57bdd8a.png)

**Comment by nicolo-ribaudo** [comment]
I will add the option to the UI as a follow up to https://github.com/babel/website/pull/1816

**Comment by oMaten** [comment]
@nicolo-ribaudo Thanks for reply

**Comment by zakhenry** [comment]
@nicolo-ribaudo has this been implemented separately? I'd like to try the pipeline operator out :)

**Comment by nicolo-ribaudo** [comment]
Yes! If you enable the stage-1 preset in the repl it should appear a dropdown where you can choose "smart" pipelines. Very soon it will also support F# pipelines.
